pkgs/by-name/ds/dsf2flac/package.nix

```
@@ -38,16 +38,16 @@ stdenv.mkDerivation {
   enableParallelBuilding = true;
 
   preConfigure = ''
-    export LIBS="$LIBS -lz"
+    export LIBS="$LIBS -lz -lboost_timer"
   '';
 
   configureFlags = [ "--with-boost-libdir=${boost.out}/lib" ];
 
-  meta = with lib; {
+  meta = {
     description = "DSD to FLAC transcoding tool";
     homepage = "https://github.com/hank/dsf2flac";
-    license = licenses.gpl2;
-    maintainers = with maintainers; [ artemist ];
+    license = lib.licenses.gpl2;
```

```suggestion
    license = lib.licenses.gpl2Plus;
```

`lib.licences.gpl2` is deprecated, see lib/licenses.nix.